test/y2partitioner/actions/resize_partition_test.rb

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "../test_helper"
+require "y2partitioner/actions/resize_partition"
+require "y2partitioner/dialogs/partition_resize"
+require "y2partitioner/device_graphs"
+
+describe Y2Partitioner::Actions::ResizePartition do
+  using Y2Storage::Refinements::SizeCasts
+
+  before do
+    allow(Yast::Wizard).to receive(:OpenNextBackDialog)
+    allow(Yast::Wizard).to receive(:CloseDialog)
+  end
+
+  subject(:action) { described_class.new(partition) }
+
+  context "With a mixed partition setup" do
```

Comment:
Is this context really needed? It looks to me like it adds not useful information and its only effect is to indent everything one level.

Reply:
Yes, I agree.